### PR TITLE
Replace simplebar with tt-react-custom-scrollbar

### PR DESCRIPTION
**Pull request by MahboobehMohammadi**

### CODE REVIEW

File: src/store/CategoricalDisplayStore.js [hunk 1]
```diff
@@ -272,11 +272,12 @@ export default class CategoricalDisplayStore {
     }
 
     @action.bound setScrollPanel(element) {
-        this.scrollPanel =  element.querySelector('.simplebar-content');
+        this.scrollPanel =  element;
```
Comment (shayanbinary) [inline src/store/CategoricalDisplayStore.js]: @MahboobehMohammadi `element` maybe `null` in some case, please handle it also

File: src/store/CategoricalDisplayStore.js [hunk 2]
```diff
@@ -285,5 +286,6 @@ export default class CategoricalDisplayStore {
         ResultsPanel: this.ResultsPanel,
         FilterPanel: this.FilterPanel,
         SearchInput: this.SearchInput,
+        isMobile: this.mainStore.chart.isMobile,
```
Comment (shayanbinary) [inline src/store/CategoricalDisplayStore.js]: maybe it's better to get chart as a `getter` like `get chart(){return this.mainStore.chart}`

File: src/components/DrawTools.jsx [hunk 3]
```diff
@@ -24,7 +24,10 @@ const DrawTools = ({
 
         <DrawToolsMenu.Body>
             <div className="body">
-                <SimpleBar
+                <Scrollbars
+                    autoHide
+                    autoHeight
+                    autoHeightMax={260}
```
Comment (shayanbinary) [inline src/components/DrawTools.jsx]: @MahboobehMohammadi  perhaps it's better to handle `max-height` in the CSS

File: src/components/PriceLine.jsx [hunk 4]
```diff
@@ -26,9 +26,9 @@ class PriceLine extends Component {
                 className={`chart-line horizontal ${draggable ? 'draggable' : ''} ${isDragging ? 'dragging' : ''} ${className || ''}`}
                 style={{ top, zIndex, color: draggable ? color : '#FFF', borderColor: color }}
                 ref={setDragLine}
-                hidden={visible ? undefined : 'true'}
-                uncentered={uncentered ? 'true' : undefined}
-                off-screen={offScreen ? 'true' : undefined}
+                hidden={visible ? undefined : true}
+                uncentered={uncentered ? true : undefined}
+                off-screen={offScreen ? true : undefined}
```
Comment (kellybinary) [inline src/components/PriceLine.jsx]: i think this can be simplified to `true || undefined`

File: src/store/CategoricalDisplayStore.js (hunk 1 above)
Comment (MahboobehMohammadi) [reply src/store/CategoricalDisplayStore.js]: @shayanbinary in which cases it might be null?

**Pull request merged**